From 5a56bfe8d281250a1deee0d116a9fcde65b9c29a Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Fri, 15 Jan 2021 18:37:05 +0000
Subject: [PATCH] IWYU: add various missing includes

std::weak_ptr and std::shared_ptr require map
*int*_t types require cstdint
---
 third_party/dawn/src/dawn_wire/client/Device.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/third_party/dawn/src/dawn_wire/client/Device.h b/third_party/dawn/src/dawn_wire/client/Device.h
index 3f16700..1082549 100644
--- third_party/dawn/src/dawn_wire/client/Device.h
+++ third_party/dawn/src/dawn_wire/client/Device.h
@@ -22,7 +22,9 @@
 #include "dawn_wire/client/ApiObjects_autogen.h"
 #include "dawn_wire/client/ObjectBase.h"
 
+#include <cstdint>
 #include <map>
+#include <memory>
 
 namespace dawn_wire { namespace client {
 
-- 
2.26.2

